io.enpass.Enpass.yaml

```diff
@@ -0,0 +1,101 @@
+# io.enpass.Enpass.yaml
+app-id: io.enpass.Enpass
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: enpass
+tags: [proprietary]
+separate-locales: false
+finish-args:
+    # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+    # Notification access
+  - --talk-name=org.freedesktop.Notifications
+    # System Tray Icon
+  - --talk-name=com.kde.StatusNotifierWatcher
+    # Secret Service
+  - --own-name=org.freedesktop.secrets
+    # Favicon download
+  - --share=network
+    # Required to save vaults in default location <$HOME/Documents>
+  - --filesystem=xdg-documents
+modules:
+  - name: enpass
+    buildsystem: simple
+    build-options:
+      env:
+        FLATPAK_ID: io.enpass.Enpass
+    build-commands:
+      - |
+
+        mkdir -p ${FLATPAK_DEST}/bin/extra
+        mkdir -p ${FLATPAK_DEST}/share/app-info/icons/flatpak/128x128
+        mkdir -p ${FLATPAK_DEST}/share/mime/packages
+
+        install -Dm 755 apply_extra --target-directory ${FLATPAK_DEST}/bin/
+        chmod +x ${FLATPAK_DEST}/bin/apply_extra
+
+        install -Dm 755 enpass --target-directory ${FLATPAK_DEST}/bin/
+        chmod +x ${FLATPAK_DEST}/bin/enpass
+
+        install -Dm 644 ${FLATPAK_ID}.desktop --target-directory ${FLATPAK_DEST}/share/applications/
+        install -Dm 644 ${FLATPAK_ID}.metainfo.xml --target-directory ${FLATPAK_DEST}/share/metainfo/
+        install -Dm 644 ${FLATPAK_ID}.mime.xml --target-directory ${FLATPAK_DEST}/share/mime/packages/
+
+        install -Dm 644 ${FLATPAK_ID}.png --target-directory ${FLATPAK_DEST}/share/app-info/icons/flatpak/128x128/
```

Comment:
```suggestion
        install -Dm 644 ${FLATPAK_ID}.png --target-directory ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/
```

Reply:
If I may ask, does the build process automatically place images in the `*/app-info/icons/flatpak/128x128` directory?

Previously, [my builds were failing because the buildbot would report it was missing](https://buildbot.flathub.org/#/builders/37/builds/3300/steps/14/logs/stdio):

```
builddir/*/share/app-info/icons/flatpak/128x128/io.enpass.Enpass.png
```

I already export the applications icons [here](https://github.com/ShobuPrime/flathub/blob/9cbe7f468a0fcedc068dfc60c55cc58219503d44/io.enpass.Enpass.yaml#L78) in the extra data.

---

It looks like it does because the output of the local build says the following after trying your change locally:

```
Saving icon /app/share/app-info/icons/flatpak/64x64/io.enpass.Enpass.png
Saving icon /app/share/app-info/icons/flatpak/128x128/io.enpass.Enpass.png
```

This must be why you said to [never do anything to the directory](https://github.com/flathub/flathub/pull/3991#discussion_r1139599594)?

Reply:
[Removed PNG](https://github.com/flathub/flathub/pull/3991/commits/7c5511e7bced4827605700be6d445510953d03cf)